doc/guides/security_release_process.md

```diff
@@ -2,118 +2,97 @@
-  [cve_management_process](https://github.com/nodejs/node/blob/master/doc/guides/cve_management_process.md).
+* [ ] Get release team volunteers for all affected lines:
+  * v12.x: ***NAME of RELEASER(S)***
+  * ... other lines, if multiple releasers
 
-* [ ] Co-ordinate with the Release team members to line up one or more releasers
-  to do the releases on the agreed date. Releaser: ***NAME of RELEASER(S)***
+## Announcement (one week in advance of the planned release)
 
-* [ ] Prep for the security announcements by getting agreement on drafts (use
-  previously announced releases as the template):
-  * pre-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
-  * post-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
+* [ ] Check that all vulnerabilities are ready for release integration:
+  * PRs against all affected release lines or cherry-pick clean
+  * Approved
+  * Pass `make test`
+  * Have CVEs
+  * Described in the pre/post announcements
 
-## Announcement (one week in advance of the planned release)
+* [ ] Pre-release announcement [email][]: ***LINK TO EMAIL***
+  (Get access from existing manager: Ben Noordhuis, Rod Vagg, Michael Dawson)
 
-* [ ] Send pre-release announcement to
-  https://groups.google.com/forum/#!forum/nodejs-sec.
-  One of the existing managers can give access (Ben
-  Noordhuis, Rod Vagg, Michael Dawson). ***LINK TO EMAIL***
-
-* [ ] Post pre-release announcement in vulnerabilities section of Nodejs.org
-  blog (https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability).
-  Use last pre-release announcement as a template (it includes blog metadata
-  such as updates to the banner on the Node.js website to indicate security
-  releases are coming).  Submit PR and land immediately. Text was already
-  reviewed in security repo.  ***LINK TO BLOG PR AND POST***
-
-* [ ] Open an issue in the build working repository with a notification of the
-  date for the security release.  Use this issue to co-ordinate with the build
-  team to ensure there will be coverage/availability of build team resources the
-  day of the release. Those who volunteer from the build WG should be available
-  in node/build during the release in case they are needed by the individual
-  doing the release. ***LINK TO BUILD ISSUE***
+* [ ] Pre-release announcement to nodejs.org blog: ***LINK TO BLOG***
+  (Re-PR the pre-approved branch from nodejs-private/nodejs.org-private to
+  nodejs/nodejs.org)
+
+* [ ] Request releaser(s) to start integrating the PRs to be released.
+
+* [ ] Notify [docker-node][] of upcoming security release date: ***LINK***
 
 ## Release day
 
+* [ ] [Lock CI](https://github.com/nodejs/build/blob/master/doc/jenkins-guide.md#before-the-release)
+> FIXME: Whoever is doing the Node.js release should have sufficient privs to
+> do this
+
 * [ ] The releaser(s) run the release process to completion.
+> FIXME: this process is a bit different from the normal release process
```

Comment:
@BethGriggs You found the process a bit different in the private/sec release from the public one, any suggestions on what should be documented, and where? Also, would it be reasonable to add the lock/unlock to release process, so whoever is doing it isn't blocked on authority?

Reply:
I think https://github.com/nodejs/node/blob/master/doc/releases.md would be a good place to incorporate the security-specific steps for the releaser. (We already have the step of adding "This is a security release." to the changelog listed in that doc.)